client/modules/ftml-wasm/tests/ftml.ts

```diff
@@ -1,191 +1,184 @@
+    assert.deepEqual(lib.parse(str) as any, {
+      "ast": {
+        "elements": [
+          {
+            "element": "container",
+            "data": {
+              "type": "paragraph",
+              "elements": [
+                {
+                  "element": "container",
+                  "data": {
+                    "type": "italics",
+                    "elements": [
+                      {
+                        "element": "text",
+                        "data": "1"
+                      }
+                    ],
+                    "attributes": {}
+                  }
                 }
-              }
-            ],
-            "attributes": {}
+              ],
+              "attributes": {}
+            }
+          },
+          {
+            "element": "footnote-block",
+            "data": {
+              "title": null,
+              "hide": false
+            }
           }
-        },
-        {
-          "element": "footnote-block",
-          "data": {
-            "title": null,
-            "hide": false
-          }
-        }
-      ],
-      "styles": [],
-      "table-of-contents": [],
-      "footnotes": []
-    },
-    "warnings": []
+        ],
+        "styles": [],
+        "table-of-contents": [],
+        "footnotes": []
+      },
+      "warnings": []
+    })
   })
-})
-
-FTML("renderHTML", async () => {
-  await lib.loading
-  const str = "//1//"
-  // we won't test the `meta` property because
-  // that property is a bit too dynamic to easily test. (it has version info in it)
-  const { html, styles } = lib.renderHTML(str)
 
-  assert.is(html, "<p><em>1</em></p>")
-  assert.is(styles.join(""), "")
-
-  // assert.equal(lib.render(str), {
-  //   html: "<p><em>1</em></p>",
-  //   meta: [
-  //     {
-  //       tag_type: "http-equiv",
-  //       name: "Content-Type",
-  //       value: "text/html"
-  //     },
-  //     { tag_type: "name", name: "generator", value: "ftml 0.7.0" },
-  //     { tag_type: "name", name: "description", value: "" },
-  //     { tag_type: "name", name: "keywords", value: "" }
-  //   ],
-  //   style: ""
-  // })
-})
+  it("renderHTML", async () => {
+    await lib.loading
+    const str = "//1//"
+    // we won't test the `meta` property because
+    // that property is a bit too dynamic to easily test. (it has version info in it)
+    const { html, styles } = lib.renderHTML(str)
+
+    assert.equal(html, "<p><em>1</em></p>")
+    assert.equal(styles.join(""), "")
+
+    // assert.equal(lib.render(str), {
+    //   html: "<p><em>1</em></p>",
+    //   meta: [
+    //     {
+    //       tag_type: "http-equiv",
+    //       name: "Content-Type",
+    //       value: "text/html"
+    //     },
+    //     { tag_type: "name", name: "generator", value: "ftml 0.7.0" },
+    //     { tag_type: "name", name: "description", value: "" },
+    //     { tag_type: "name", name: "keywords", value: "" }
+    //   ],
+    //   style: ""
+    // })
```

Comment:
Remove old commented code?

Reply:
iirc I left this here to show what testing for the meta field looked like, but I can probably just remove this.

Reply:
I think a more intentional comment would be better at that.